src/main/java/domain/LottoMachine.java

```diff
@@ -1,12 +1,11 @@
+package domain;
+
+import static domain.LottoInformation.*;
+
 public class LottoMachine {
 
     private final int ticket;
 
-    private static final int LOTTO_PRICE = 1000;
-    private static final int LOTTO_NUMBER_START = 1;
-    private static final int LOTTO_NUMBER_END = 45;
-    private static final int LOTTO_NUMBER_COUNT = 6;
-
     public LottoMachine(Price price) {
         this.ticket = price.getValue() / LOTTO_PRICE;
```

Comment:
1. price 객체끼리 계산을 하도록 구현하는것은 어떻게 생각하시나요?

```suggestion
    public LottoMachine(Price price) {
        this.ticket = price.divide(Price.A_LOTTO_TICKET_PRICE);
```

2. ticket 이라는 변수는 티켓의 수 라는 의미를 충분히 표현하고 있을까요?
3. 로또 기계가 ticket 수 라는 상태를 가져야할까요?
  - ticket수를 밖에서 계산하고 LottoMachine의 generateLottos 메서드 인자로 ticket 수를 넘기는 방법
  - LottoMachine의 generateLottos 메서드에 지불금액을 넘기는 방법

Reply:
1.  금액들의 나눗셈에 관한 단순 계산은 Price가 가져도 될 것 같네요! 좋은 생각하게 해주셔서 감사합니다!

2. 저는 충분하다고 생각했는데,찰리가 의문을 주신거 보면 아닌 것 같네요! 처음 코드를 보는 사람입장에서는 단순히 티켓이라고만 되었다면 헷갈릴 것 같네요. ticketCount로 변수명을 바꿔보려 하는데 찰리 생각은 어떠신가요? 

3. 로또 머신이 로또를 만들어내는 역할을 담당하고 있는데, 몇 개의 로또를 만들지를 스스로가 정해야한다고 생각해서 안에서 직접 구현하였습니다. 하지만 ticket 이라는 수 의 상태까지 가져야 할지에 대해서는 깊게 생각해보지 못했네요,  티켓 개수에 관한 변수를 누군가 가지고 있어야 생각했는데 꼭 그렇지 않은 것 같네요. 앞서 Price에 Divide 메서드를 활용해서 티켓 개수를 반환하는 메서드를 추가하여 LottoMachine generateLottos에 메서드 인자로 넘겨보도록 수정해보았습니다!

Reply:
> 저는 충분하다고 생각했는데,찰리가 의문을 주신거 보면 아닌 것 같네요! 처음 코드를 보는 사람입장에서는 단순히 티켓이라고만 되었다면 헷갈릴 것 같네요. ticketCount로 변수명을 바꿔보려 하는데 찰리 생각은 어떠신가요?

ticketCount 명확한 이름이라고 생각해요 🤗